src/pages/Books.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Book from '../components/book';
+import './books.css';
+import AddBook from '../components/addBook';
+
+const Books = () => (
+  <div className="books">
+    <ul className="ulBooks">
+      <li className="book"><Book genre="Action" title="The Hunger Games" author="Suzanne Collins" /></li>
+      <li className="book"><Book genre="Science Fiction" title="Dune" author="Frank Herbert" /></li>
+      <li className="book"><Book title="Capital in the Twenty-First Century" author="Suzanne Collins" /></li>
```

Comment:
- [ ] Kindly implement an array of books and then use `map` to iterate of the list, instead of hard coding 3 book components, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=display%20the%20list%20of%20books) requirement.